.ci/pipeline_definitions

```diff
@@ -58,7 +59,7 @@ terraformer:
               PROVIDER: slim
     steps:
       verify:
-        image: 'eu.gcr.io/gardener-project/3rd/golang:1.16.2'
+        image: 'golang:1.16.2'
```

Comment:
Hmm, is the verify step also using a different runtime/config/environment that is not susceptible to docker hub rate limits? Why are we now able to switch back?

Reply:
@timebertt : I think for the build-steps, this should not be such a big issue, as concourse does cache the images.

the actual quota-issues stemmed from the image-builds. With the switch to kaniko-builder, however, we have the possibility to authenticate against all required source registries, so this issue should be a relict from the past :-)